src/Symfony/Component/Console/Output/Output.php

```diff
@@ -140,7 +140,8 @@ public function writeln($messages, $options = self::OUTPUT_NORMAL)
     public function write($messages, $newline = false, $options = self::OUTPUT_NORMAL)
     {
         $messages = (array) $messages;
-
+        $options = is_numeric($options) ? (int) $options : 0;
```

Comment:
This is just one of the options for bypassing exception "A non-numeric value encountered" in PHP 7.1
It is also possible such as:
$options = 'ERROR' == $options ? 0 : $options;